wfn/matching.go

```diff
@@ -28,19 +28,20 @@ const (
 
 // HasWildcard returns true if attribute has a wildcard symbol in it
 func HasWildcard(s string) bool {
```

Comment:
have you thought about doing this however?
https://play.golang.org/p/La0qbCQs9GR

Reply:
Briefly, I didn't think it can possibly be faster. The version in playground you linked has a bug, it returns false for something like `bar\\?` (the backslash itself is quoted, the question mark isn't as a result, so it should return true). But even disregarding this (your version has 2 suffix):
```
goos: linux
goarch: amd64
pkg: github.com/facebookincubator/nvdtools/wfn
BenchmarkHasWildcard/has-4              75515116                15.9 ns/op
BenchmarkHasWildcard/has_not-4          33865971                31.6 ns/op
BenchmarkHasWildcard/has_escaped-4      45392180                26.2 ns/op
BenchmarkHasWildcard2/has-4             12806960                88.3 ns/op
BenchmarkHasWildcard2/has_not-4         11228508               101 ns/op
BenchmarkHasWildcard2/has_escaped-4             14998665                75.9 ns/op
PASS
ok      github.com/facebookincubator/nvdtools/wfn       8.173s
```